OneMore/Commands/Edit/ConvertMarkdownCommand.cs

```diff
@@ -97,8 +99,9 @@ public override async Task Execute(params object[] args)
 
 				converter
 					.RewriteHeadings(touched)
-					.RewriteTodo(touched)
-					.SpaceOutParagraphs(touched, 12);
+					.RewriteTodo(touched);
+// disabled, as it will space out also short lines. Added space in heading defintion instead
```

Comment:
See my note below about changing the heading definitions. They need to match the OneNote built-in definitions. This needs to be solved a diffferent way

Reply:
Moved the spacebefore/after handling into RewriteHeadings